phpBB/memberlist.php

```diff
@@ -1345,13 +1347,19 @@
 			);
 		}
 
+		$user_types = array(USER_NORMAL, USER_FOUNDER);
+		if ($auth->acl_get('a_user') || $user->data['user_type'] == USER_FOUNDER)
```

Comment:
the check on the user type shouldn't be necessary. It is already enforced in the auth class: https://github.com/phpbb/phpbb/blob/3.1.x/phpBB/phpbb/auth/auth.php#L424